File: public/phase-2-20160620022203/C_SHARP/snowflakes.cs
// Name : Snowflakes
using System;
namespace Application {
	class snowflakes {
		public static void Main(string[] args) {
		// input:
		// n : length of array
		// k : number of  letters of the English alphabet
		// english_letters :  contains n uppercase English letters
		string[] elements = (Console.ReadLine()).Split(' ');
		int n = Convert.ToInt32(elements[0]);
		int k = Convert.ToInt32(elements[1]);
		string english_letters = Console.ReadLine();


		// write your code here
		// store your results in `min_number_of_changes`,`changed_array`

		// output
		// Dummy Data
		int min_number_of_changes=2;
		string changed_array="ABCACA";

		Console.WriteLine(min_number_of_changes);
		Console.WriteLine(changed_array);

		}	
	}
}

File: public/phase-2-20160620022203/C/snowflakes.c
// Name : Snowflakes
#include <stdio.h>
int main() {
	// input:
	// n : length of array
	// k : number of  letters of the English alphabet
	// english_letters :  contains n uppercase English letters
	int n, k, min_number_of_changes=2;
	char english_letters[105], changed_array[105]="ABCACA";
	scanf("%d %d", &n, &k);
	scanf("%s", &english_letters);


	// write your code here
	// store your results in `min_number_of_changes`,`changed_array`

	// output
	printf("%d", min_number_of_changes);
	printf("%s", changed_array);
	return 0;
}